src/javascript/EditPanel/header/ContentBreadcrumb/ContentPath/ContentPath.container.jsx

```diff
@@ -23,10 +25,22 @@ const ContentPathContainer = ({path}) => {
     });
 
     const handleNavigation = path => {
+        let newMode = 'content-folders';
+
+        if (path.indexOf('/files') !== -1) {
+            newMode = 'media';
```

Comment:
This test is not enough as a content named `filesName` will match files even not being under `/files`.
You need to do the lookup in the right place in the path `/sites/sitename/${type}/`
and not forget about the trailing `/` 

Also the default is pages, as we know that contents are under `contents` and we can have other pages at the same level as `home`

Reply:
All the cases describe above are not fulfilled. With the current code a path like `/sites/siteKey/files` will be resolved as `pages` instead of `media`